### PR TITLE
Copyediting for Dashboard

### PR DESCRIPTION
**Pull request by jefferyjohn**
# Description 

Add additional card for Installation, miscellaneous copyediting and formatting.

# Screenshots

image

### CODE REVIEW

File: src/pages/getting_started.tsx [hunk 1]
```diff
@@ -1,46 +1,55 @@
...
     <Card className="mb-3">
       <Card.Body>
-        <Card.Title>Getting Started</Card.Title>
+        <Card.Title>Installation</Card.Title>
         <Card.Text>
-          Follow through the instructions below to learn more about installation
-          instructions, how you can upload your bot submissions, request
-          scrimmages with other players, and check out your match results!
+          Visit our AWAP 2023{" "}
+          <a href="https://github.com/ACM-CMU/awap-viewer-2023">Viewer</a> and{" "}
```
Comment (victorhuangwq) [inline src/pages/getting_started.tsx:18]: they don't have to install viewer. they just need to go to view.awap.acmatcmu.com

File: src/pages/getting_started.tsx [hunk 2]
```diff
@@ -1,46 +1,55 @@
...
+          <a href="https://github.com/ACM-CMU/awap-engine-2023">Engine</a>{" "}
+          GitHub repos to install the Martian environment. You'll need Python
```
Comment (victorhuangwq) [inline src/pages/getting_started.tsx:20]: Maybe we should link the game guide here as well.

**Review by victorhuangwq — CHANGES_REQUESTED** [review]
Some changes requested.

File: src/pages/getting_started.tsx (hunk 1 above)
Comment (jefferyjohn) [reply src/pages/getting_started.tsx:18]: fixed - will they need to install engine?

File: src/pages/getting_started.tsx (hunk 2 above)
Comment (jefferyjohn) [reply src/pages/getting_started.tsx:20]: added!

**Comment by jefferyjohn** [comment]
Latest commit
image

**Comment by victorhuangwq** [comment]
They won't need NodeJS @jefferyjohn

File: src/pages/getting_started.tsx (hunk 1 above)
Comment (victorhuangwq) [reply src/pages/getting_started.tsx]: they will need to install python and the relevant packages to run the engine

**Pull request merged**